contributing.md

```diff
@@ -58,13 +56,13 @@ Examples of minor corrections include:
 
 The Working Group uses pull requests to track changes. To submit a change to the specification:
 
-**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension)
+**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed the creation of a PR based on a particular Issue.
 
 **3.2.** Modify the Specification to Address the Issue. This work can happen outside of the GitHub flow as you and your team may want to do live collaboration on the spec content before converting to GitHub and Markdown. Once mostly aligned within your group, you can convert the content to markdown and submit the change to the topic branch created above.
 
 **3.3.** Submit a Pull Request to merge the change from the topic branch to the 'working_draft' branch (see details below)
 
-**3.4.** Notify the focus members about your pull request and ask for feedback. Address comments on your topic branch until reviewers have approved.
+**3.4.** Notify the Maintainers about your pull request and ask for feedback. Address comments on your topic branch until reviewers have approved.
```

Comment:
I think when PR's go from WIP to normal, members should be notified. There's no reason it has to be routed through maintainers only since 1. it causes delays 2. it's already an approved work item. If it's not something in scope for the current milestone, it shouldn't move out of WIP and likely should not even be a PR. We need the Issue to be the place to drive collaboration and support (and get triaged and prioritized by maintainers).

Reply:
Thanks, Udam. I see your point. I had a different reading. I will reverse the change.